example/data/input/pipeline/datamappingpipeline.json

```diff
@@ -0,0 +1,320 @@
+{
+  "__class__":"com.northconcepts.datapipeline.foundations.pipeline.DataMappingPipeline",
+  "name":null,
+  "description":null,
+  "input":{
+    "__class__":"com.northconcepts.datapipeline.foundations.pipeline.input.CsvPipelineInput",
```

Comment:
Can you update EntityDef in DPF to exclude `__class__` like we do for FieldDef since I don't think we need it -- it's not a super type of anything?

Here's a new card for this work: https://northconcepts.atlassian.net/browse/DP-1622

Reply:
done